RHEL/7/templates/static/oval/sshd_use_priv_separation.xml

```diff
@@ -23,7 +23,7 @@ root privileges when not needed.</description>
   </ind:textfilecontent54_test>
   <ind:textfilecontent54_object id="obj_sshd_use_priv_separation" version="1">
     <ind:filepath>/etc/ssh/sshd_config</ind:filepath>
-    <ind:pattern operation="pattern match">^[\s]*(?i)UsePrivilegeSeparation(?-i)[\s]+yes[\s]*(?:|(?:#.*))?$</ind:pattern>
+    <ind:pattern operation="pattern match">^[\s]*(?i)UsePrivilegeSeparation(?-i)[\s]+(yes|sandbox)[\s]*(?:|(?:#.*))?$</ind:pattern>
```

Comment:
Probably should have both as (might be wrong on this) but I *think* that RHEL6 does not have sandbox mode. I'd say that this PR gets updated to use a variable that is then configured in the profiles.